pkg/meta/autoid/autoid_service.go

```diff
@@ -72,8 +73,15 @@ func NewClientDiscover(etcdCli *clientv3.Client) *ClientDiscover {
 	}
 }
 
+func getLeaderEtcdPath(keyspaceID uint32) string {
+	if keyspaceID == uint32(tikv.NullspaceID) {
+		return autoIDLeaderPath
+	}
+	return "/" + autoIDLeaderPath
```

Comment:
the only difference is `/` in the two cases? I suppose `keyspaceID` should be encoded to path

Reply:
This change is mainly because the keyspace scenario requires adding a '/' in front of the autoIDLeaderPath. 

Otherwise, in the case where keyspaceID = 1999, 
the key concatenation will become `/keyspaces/tidb/1999tidb/autoid/leader`.

 What is actually expected is `/keyspaces/tidb/1999/tidb/autoid/leader`.

Reply:
The keyspaceID has already been added to the key during the domain initialization by setting the etcd namespace